src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -257,7 +260,8 @@ private void RecoverViaExtPubKey(string walletName, string extPubKey, int accoun
             {
                 ExtPubKey = extPubKey,
                 AccountIndex = accountIndex,
-                Name = walletName
+                Name = walletName,
+                CreationDate = DateTime.Now
```

Comment:
why the change?

Reply:
It fixes the failing `CreateExtPubKeyOnlyWallet_creates_wallet_with_extra_flag` test that no one else seemed to have bothered about yet.

Reply:
You mean this test was failing before that change?

Reply:
As far as I am aware, yes. `AddNewAccount_for_xpub_only_wallet_informs_user_to_create_new_wallet_` as well.

Reply:
These 2 don't fail for me or for the others. The integration tests are not running smoothy and sometimes a rerun of the failed ones fixes the issue.

Reply:
Because the creationDate used is the default one: 01/01/0001 and doesn't fail.

Reply:
It does default, but it still throws:

```
{System.ArgumentOutOfRangeException: The UTC time represented when the offset is applied must be between year 0 and 10,000.
Parameter name: offset
   at System.DateTimeOffset.ValidateDate(DateTime dateTime, TimeSpan offset)
   at System.DateTimeOffset..ctor(DateTime dateTime)
   at System.DateTimeOffset.op_Implicit(DateTime dateTime)
   at Stratis.Bitcoin.Features.Wallet.WalletManager.RecoverWallet(String name, ExtPubKey extPubKey, Int32 accountIndex, DateTime creationTime) in C:\StratisBitcoinFullNode\src\Stratis.Bitcoin.Features.Wallet\WalletManager.cs:line 369
   at Stratis.Bitcoin.Features.Wallet.Controllers.WalletController.RecoverViaExtPubKey(WalletExtPubRecoveryRequest request) in C:\StratisBitcoinFullNode\src\Stratis.Bitcoin.Features.Wallet\Controllers\WalletController.cs:line 285}
```